jekyll/_cci2/local-cli.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: classic-docs
+title: "The CircleCI Local Command Line Interface (CLI)"
+short-title: "Using the CircleCI Local CLI"
+description: "How to run local jobs with the CLI"
+categories: [troubleshooting]
+order: 10
+alias: /docs/2.0/local-jobs/
+---
+
+*[Build Environments]({{ site.baseurl }}/2.0/build/) > {{ page.title }}*
+
+* TOC
+{:toc}
+
+## Overview
+
+The `circleci` commands enable you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase. 
+
+You can also run `circleci` commands in your `config.yml` file for jobs that use the primary container image. This is particularly useful for globbing or splitting tests among containers. 
+
+**Note:** The Local CLI does not yet support Workflows.
+
+## Installing
+
+### Ubuntu 16.04+, Fedora 27+, and other Distros Supporting Snap Packages
+
+```
+sudo snap install docker circleci
+sudo snap connect circleci:docker docker
+```
+
+### macOS and other Linux Distros
+
+1. Install and configure Docker by using the [docker installation instructions](https://docs.docker.com/engine/installation/).
+
+2. To install the CLI, run the following command:
+
+```
+$ curl -o /usr/local/bin/circleci https://circle-downloads.s3.amazonaws.com/releases/build_agent_wrapper/circleci && chmod +x /usr/local/bin/circleci
+```
+
+The CLI, `circleci`, is downloaded to the `/usr/local/bin` directory. If you do not have write permissions for `/usr/local/bin`, you might need to run the above commands with `sudo`. The CLI automatically checks for updates and will prompt you if one is available. 
+
+### Validating 2.0 YAML Syntax
+
+1. Type `circleci` with one of six available commands (`build`, `config`, `help`, `step`, `tests`, and `version`) followed by a valid flag. For example:
+```
+$ circleci config validate -c .circleci/config.yml
+```
+The config validate command checks your local config.yml file for syntax errors.
```

Comment:
"The config validate command" --> `circleci config validate`

I'd just backtick the actual command instead of talking around it.

Reply:
This pre-existed my change. I can do this though if you'd like (I agree with you).

Reply:
Yeah, this is why I stopped the review — when I realized I was commenting on preexisting content. No worries on changing this.